app/src/main/java/com/gh4a/adapter/timeline/ReviewViewHolder.java

```diff
@@ -235,7 +235,10 @@ private void formatTitle(final Review review) {
                 break;
         }
 
-        String login = review.user().login();
+        User reviewer = review.user();
+        // In this case, GitHub API returns a null user (instead of the "ghost" user) when
+        // the review author is a deleted user
+        String login = reviewer != null ? reviewer.login() : "ghost";
```

Comment:
I think using `ApiHelpers.getUserLogin()` would be more appropriate.

Reply:
Done, thanks! I wasn't aware of that method